src/notelistview.cpp

```diff
@@ -929,14 +929,17 @@ QPixmap NoteListViewPrivate::renderToPixmap(const QModelIndexList &indexes, QRec
         option.rect = paintPairs.at(j).rect.translated(-r->topLeft());
         const QModelIndex &current = paintPairs.at(j).index;
 #endif
+        Q_Q(const QAbstractItemView);
         adjustViewOptionsForIndex(&option, current);
-        delegateForIndex(current)->paint(&painter, option, current);
+        q->itemDelegateForIndex(current)->paint(&painter, option, current);
     }
     return pixmap;
 }
 
 QStyleOptionViewItem NoteListViewPrivate::viewOptionsV1() const
 {
     Q_Q(const NoteListView);
-    return q->viewOptions();
+    QStyleOptionViewItem* option = new QStyleOptionViewItem();
+    q->initViewItemOption(option);
+    return *option;
```

Comment:
Don't know what this code does but it builds :)

Reply:
But it allocates a `QStyleOptionViewItem` on the heap, which is copied before the function returns and then leaked. This temporary object should be placed on the stack:
```c++
    QStyleOptionViewItem option;
    q->initViewItemOption(option);
    return option;
```